src/components/Person/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = (props) => {
+  const {
+    name,
+    age = null,
+    sex,
+    isMarried = false,
+    partnerName = null,
```

Comment:
This values wouldn't be checked against propTypes, let's use `defaultProps` instead